docs/technical/new_instance.md

```diff
@@ -0,0 +1,188 @@
+# Creating a new Insights instance
+The overall idea is strong isolation across districts, which means some initial setup overhead.  Here's how you do it!  More detailed notes for various steps are below.
+
+### 1: Stand up an instance
+Goal: Visit the home page signed-out
```

Comment:
I like the Goals for each section and the checklist structure a lot! 

The checklist structure makes me wonder if it would be nice to make these markdown checkboxes — `[ ]` — so we could copy these lists for each district and and tick off each step as we go. 

The Goal for each one makes me think that each section could be accompanied by a more detailed guide down the line if that's needed.

Reply:
Yeah I think of this more as a "remember what we did for New Bedford?" aid for you and I than something that we could copy-paste and follow super literally.  We don't do this process often enough that we really have to optimize this, so this is more about "capture the list of things we just did" than "make a how-to doc that a new person could just follow."

And next time we do this, it seems better to incrementally automate parts of this (like your "make a new Heroku app" script).